fs/report_test.go

```diff
@@ -207,3 +207,70 @@ func TestMatchFileContent(t *testing.T) {
 		assert.Equal(t, result.(cmpFailure).FailureMessage(), expected)
 	})
 }
+
+func TestMatchExtraFilesGlob(t *testing.T) {
+	dir := NewDir(t, t.Name(),
+		WithFile("t.go", "data"),
+		WithFile("a.go", "data"),
+		WithFile("conf.yml", "content", WithMode(0600)))
+	defer dir.Remove()
+
+	t.Run("matching globs", func(t *testing.T) {
+		manifest := Expected(t,
+			MatchFilesWithGlob("*.go", MatchAnyFileMode, MatchAnyFileContent),
+			MatchFilesWithGlob("*.yml", MatchAnyFileMode, MatchAnyFileContent))
+		assert.Assert(t, Equal(dir.Path(), manifest))
+	})
+
+	t.Run("matching globs with wrong mode", func(t *testing.T) {
+		manifest := Expected(t,
+			MatchFilesWithGlob("*.go", MatchAnyFileMode, MatchAnyFileContent),
+			MatchFilesWithGlob("*.yml", MatchAnyFileContent, WithMode(0700)))
+
+		result := Equal(dir.Path(), manifest)()
+
+		if runtime.GOOS == "windows" {
+			assert.Assert(t, result.Success())
```

Comment:
I guess this is because of the errors here: https://ci.appveyor.com/project/gotestyourself/gotestyourself/builds/20989128 ?

I'm not sure what's up with permissions on windows. We do some windows specific stuff in `fs/manifest_windows.go` to attempt to match the default behaviour of creating a file on windows, and also (I think) to work around the fact that there is no concept of group or other permissions.

It might be better to skip the subtest on windows rather than having an opposite expectation.